aten/src/ATen/native/cuda/Loss.cu

```diff
@@ -46,32 +46,6 @@ void binary_cross_entropy_backward_out_kernel(Tensor& grad_input, const Tensor&
 
 namespace at { namespace native {
 
-Tensor kl_div_backward_cuda(const Tensor& grad, const Tensor& input, const Tensor& target, int64_t reduction, bool log_target) {
```

Comment:
These function and the `_cpu` counterpart you should keep and remove in the other PR.